src/core.js

```diff
@@ -1653,6 +1655,34 @@ Strophe.Connection.prototype = {
         }
     },
 
+    /** Function: addHTTPErrorHandler
+     *  Add callback to HTTP error
+     *
+     *  The ability to create a callback function for HTTP errors.
+     *
+     *  This handlers will fire only with http-bind service only.
```

Comment:
Grammar: `These handlers will fire only in the case of BOSH and the http-bind service.`